app/components/offering-editor.js

```diff
@@ -1,23 +1,28 @@
 import Ember from 'ember';
 import DS from 'ember-data';
 import moment from 'moment';
+import { translationMacro as t } from "ember-i18n";
```

Comment:
the translationmacro doesn't work for us right now because when we change language it doesn't change.  This is fixed in the latest version, but that breaks some tests.  You can update the package if you want and fix the broken tests, or you can just use a computed property which depends on 'i18n.locale' and then call this.get('i18n').t() to do the translation.